### PR TITLE
Add inject_timestamp configuration option

### PR DESCRIPTION
**Pull request by dyladan**
- Unit test added
- Configuration documented

### CODE REVIEW

File: lib/fluent/plugin/out_dynatrace.rb [hunk 1]
```diff
@@ -81,9 +84,10 @@ def process(_tag, es)
         log.on_trace { log.trace('#process') }
         records = 0
         # es = inject_values_to_event_stream(tag, es)
```
Comment (pirgeo) [inline lib/fluent/plugin/out_dynatrace.rb:86]: I guess this is old code? Doesn't really have to do with the PR however...

File: lib/fluent/plugin/out_dynatrace.rb [hunk 2]
```diff
@@ -92,8 +96,9 @@ def process(_tag, es)
       def write(chunk)
         log.on_trace { log.trace('#write') }
         records = []
-        chunk.each do |_time, record|
+        chunk.each do |time, record|
           # records.push(inject_values_to_record(chunk.metadata.tag, time, record))
```
Comment (pirgeo) [inline lib/fluent/plugin/out_dynatrace.rb:100]: same as above.
```suggestion
```

**Review by pirgeo — APPROVED** [review]
Looking good, just nitpicks

File: lib/fluent/plugin/out_dynatrace.rb (hunk 2 above)
Comment (dyladan) [reply lib/fluent/plugin/out_dynatrace.rb:100]: Same answer

**Review by joaopgrassi — APPROVED** [review]

**Pull request merged**